ui/src/kapacitor/components/KapacitorRules.js

```diff
@@ -54,51 +54,50 @@ const KapacitorRules = ({
 
   return (
     <PageContents source={source}>
-      <div className="panel-heading u-flex u-ai-center u-jc-space-between">
-        <h2 className="panel-title">
-          {builderHeader}
-        </h2>
-        <div className="u-flex u-ai-center u-jc-space-between">
+      <div className="panel">
+        <div className="panel-heading">
+          <h2 className="panel-title">
+            {builderHeader}
+          </h2>
+          <div className="u-flex u-ai-center u-jc-space-between">
```

Comment:
do we need these classes?

Reply:
They were removed in a bunch of other places, wondering if they should be removed here

Reply:
ah good catch, I don't think that div is actually doing anything